README.md

```diff
@@ -1,5 +1,11 @@
 # The Solidity Contract-Oriented Programming Language
-You can talk to us on [![solidity at https://gitter.im/ethereum/solidity](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/solidity?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge). Questions, feedback, and suggestions are welcome!
+
+[![Matrix Chat](https://img.shields.io/badge/Matrix%20-chat-brightgreen?style=plastic&logo=matrix)](https://matrix.to/#/#ethereum_solidity:gitter.im)
```

Comment:
I think using the `ethereum/solidity` chat here with the "talk to us" text is misleading. Only @chriseth is ever reading that chat.

Not sure what is the best option. We have some section about contribution in the docs, some other links on the website, etc. Do we want to use this README here for developers or users?

Reply:
Good question, I was actually also thinking about the fact that we are better "reachable" in the ``solidity-dev``chat.

Reply:
I just used the ``solidity``general chat because that was the one we were linking before. But I think I would tend towards rather using the ``solidity-dev``one..

Reply:
I think solidity is probably fine. I also check the chat. More experienced users would know that they can contact us on solidity-dev for more important or compiler related stuff.

Reply:
Also ``solidity-dev``is mentioned in the contributing section of the docs.

Reply:
Alright. I'll merge this then.